lib/graphql/schema/field.rb

```diff
@@ -132,6 +132,11 @@ def description(text)
             @field.description = text
           end
         end
+
+        def resolve(resolve_proc)
```

Comment:
Since this is a change in the 1.8 API, we should have a separate PR to discuss it (like @esjee did for #1158). Splitting this into two separate PRs allows us to focus on the upgrader in this PR and have a focused discussion around the API change. Please note: I don't object to this change per se, but I do want to stick to the API change process.

What do you think?

Reply:
Besides this comment, I really like the improvements on the upgrader!

Reply:
OK. I will pick it to a new PR.